addons/hr_homeworking/security/ir.model.access.csv

```diff
@@ -0,0 +1,3 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_hr_homeworking,hr.homeworking,model_hr_homeworking,base.group_user,1,1,1,1
+acces_hr_popover_delete_wizard,calendar.popover.delete.wizard.homework,model_calendar_popover_delete_wizard_homework,base.group_user,1,1,1,1
```

Comment:
Missing ending new line (as for several other files)

Ok you grand access to every internal users to the deletion wizard, but you should restrict the access to your own homeworking location as well using an ir.rule.